eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -43,6 +43,7 @@
             <configuration>
               <sources>
                 <source>..\..\applicationconfig\client\src\main\java</source>
+                <source>..\..\applicationconfig\client\src\samples\java</source>
                 <source>..\..\core\azure-core\src\main\java</source>
                 <source>..\..\core\azure-core-auth\src\main\jasva</source>
```

Comment:
Can you fix the 'jasva' typo I spotted on this line?

Reply:
Done